Core/test/ParticleTest.cpp

```diff
@@ -4,8 +4,8 @@
 
```

Comment:
You could also rename this test to FourMomentumTest (also the TestSuite thing below)

Reply:
Good one, see https://github.com/ComPWA/ComPWA/pull/310/commits/66b5656117a3a1286c270b34e7da7cb1d2a2d399